### PR TITLE
Add service example

### PR DESCRIPTION
**Pull request by pablogs9**

### CODE REVIEW

File: examples/micro-ros_addtwoints_service/micro-ros_addtwoints_service.ino [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+const rosidl_service_type_support_t * service_type_support;
+rcl_wait_set_t wait_set;
+rmw_request_id_t req_id;
+example_interfaces__srv__AddTwoInts_Response res;
+example_interfaces__srv__AddTwoInts_Request req;
+
+const char * service_name = "/add_two_ints";
+
+void setup() {
+ delay(1000);
```
Comment (jamoralp) [inline examples/micro-ros_addtwoints_service/micro-ros_addtwoints_service.ino:27]: why is this necessary?

Comment (pablogs9) [reply examples/micro-ros_addtwoints_service/micro-ros_addtwoints_service.ino:27]: Not really necessary but useful  on some platforms where USB Serial takes a bit to start
